travis/buildViaTravis.sh

```diff
@@ -17,6 +17,9 @@ if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then
   INTEGRATION_TEST_DB=postgresql ${GRADLE} ${GRADLE_OPTIONS} genie-web:integrationTest
   # Build Docker images and compile documentation
   ${GRADLE} ${GRADLE_OPTIONS} javadoc asciidoc dockerBuildAllImages
+elif [ "$TRAVIS_PULL_REQUEST" == "false" ] && [ "$TRAVIS_TAG" == "" ] && [ "$TRAVIS_TAG" == "dev-snapshot" ]; then
```

Comment:
shouldn't this be `"$TRAVIS-BRANCH" == "dev-snapshot"`?

I'd prefer to look into ways of speeding up the regular build than building in hack arounds like this. This will still cause `latest.snapshot` to end up being these artifacts even if they would have failed some check

Reply:
The purpose here is to make snapshots available for downstream testing without having to merge into master.
It's a mechanism to use in a selected set of situations (it requires a force push to a specific upstream branch).
The shortened build was just a convenience.